reactor-test/src/main/java/reactor/test/publisher/TestPublisher.java

```diff
@@ -16,16 +16,18 @@
 
 package reactor.test.publisher;
 
-import java.util.Arrays;
 import java.util.EnumSet;
 import java.util.Objects;
+import java.util.function.Consumer;
```

Comment:
As the API got reverted, the unused imports can be removed.

Reply:
done